1-js/03-code-quality/02-coding-style/article.md

```diff
@@ -269,54 +265,54 @@ If you are writing several "helper" functions and the code that uses them, there
 - [StandardJS](https://standardjs.com/)
-- (plus many more)
+- (còn nhiều hơn nữa)
 
-If you're a novice developer, start with the cheat sheet at the beginning of this chapter. Then you can browse other style guides to pick up more ideas and decide which one you like best.
+Nếu bạn là một nhà phát triển mới làm quen, hãy bắt đầu với bảng hỗ trợ ở đầu chương này. Sau đó, bạn có thể duyệt qua các hướng dẫn về phong cách khác để chọn thêm ý tưởng và quyết định xem bạn thích ý tưởng nào nhất.
 
-## Automated Linters
+## Linters tự động
 
-Linters are tools that can automatically check the style of your code and make improving suggestions.
+Linters là công cụ có thể tự động kiểm tra kiểu mã của bạn và đưa ra các đề xuất cải tiến.
 
-The great thing about them is that style-checking can also find some bugs, like typos in variable or function names. Because of this feature, using a linter is recommended even if you don't want to stick to one particular "code style".
+Điều tuyệt vời về chúng là kiểm tra kiểu cũng có thể tìm thấy một số lỗi, như lỗi chính tả trong tên biến hoặc hàm. Do tính năng này, nên sử dụng linter ngay cả khi bạn không muốn dính vào một "kiểu mã" cụ thể.
 
-Here are some well-known linting tools:
+Dưới đây là một số công cụ linting nổi tiếng:
 
-- [JSLint](http://www.jslint.com/) -- one of the first linters.
-- [JSHint](http://www.jshint.com/) -- more settings than JSLint.
-- [ESLint](http://eslint.org/) -- probably the newest one.
+- [JSLint](http://www.jslint.com/) -- một trong những công cụ đầu tiên.
+- [JSHint](http://www.jshint.com/) -- nhiều cài đặt hơn JSLint.
+- [ESLint](http://eslint.org/) -- có lẽ là cái mới nhất.
 
-All of them can do the job. The author uses [ESLint](http://eslint.org/).
+Tất cả trong số họ có thể làm công việc. Ở đây, tác giả sử dụng [ESLint](http://eslint.org/).
 
-Most linters are integrated with many popular editors: just enable the plugin in the editor and configure the style.
+Hầu hết các linters đều được tích hợp với nhiều trình chỉnh sửa phổ biến: chỉ cần bật plugin trong trình chỉnh sửa và tùy chỉnh kiểu.
 
-For instance, for ESLint you should do the following:
+Chẳng hạn, đối với ESLint, bạn nên làm như sau:
 
-1. Install [Node.js](https://nodejs.org/).
-2. Install ESLint with the command `npm install -g eslint` (npm is a JavaScript package installer).
-3. Create a config file named `.eslintrc` in the root of your JavaScript project (in the folder that contains all your files).
-4. Install/enable the plugin for your editor that integrates with ESLint. The majority of editors have one.
+1. Cài đặt [Node.js](https://nodejs.org/).
+2. Cài đặt ESLint bằng lệnh `npm install -g eslint` (npm là trình cài đặt gói JavaScript).
+3. Tạo một tệp cấu hình có tên `.eslintrc` trong thư mục gốc của dự án JavaScript của bạn (trong thư mục chứa tất cả các tệp của bạn).
+4. Cài đặt/bật plugin cho trình chỉnh sửa tích hợp với ESLint của bạn. Phần lớn các biên tập viên có một.
```

Comment:
- Nên giữ nguyên chữ editor
- The majority of editors have one => Đa số các editor đều có